node/core/runtime-api/src/cache.rs

```diff
@@ -131,6 +133,18 @@ impl RequestResultCache {
 		self.validators.put(relay_parent, validators);
 	}
 
+	pub(crate) fn minimum_backing_votes(&mut self, relay_parent: &Hash) -> Option<u32> {
+		self.minimum_backing_votes.get(relay_parent).copied()
+	}
+
+	pub(crate) fn cache_minimum_backing_votes(
+		&mut self,
+		relay_parent: Hash,
+		minimum_backing_votes: u32,
+	) {
+		self.minimum_backing_votes.put(relay_parent, minimum_backing_votes);
```

Comment:
No. This should be cached per `SessionIndex` not per relay parent as it is only supposed to change at session boundaries.